src/index.tsx

```diff
@@ -515,7 +516,7 @@ function exportAsPNG({
 
   // create a temporary <a> elem which we'll use to download the image
   const link = document.createElement("a");
-  link.setAttribute("download", "excalidraw.png");
+  link.setAttribute("download", `${fileName}`);
```

Comment:
We need to keep the .png at the end

Reply:
ishh, forgot. to think that was the reason for the template strings in the first place.